EnableIPythonDisplay.swift

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Hooks IPython to the KernelCommunicator, so that it can send display
+/// messages to Jupyter.
+
+import Python
+
+// Workaround SR-7757.
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import func Darwin.C.dlopen
+#else
+import func Glibc.dlopen
+#endif
+dlopen("libpython2.7.so", RTLD_NOW | RTLD_GLOBAL)
+
+var _socket: PythonObject = Python.None
+var _shell: PythonObject = Python.None
+
+func enableIPythonDisplay() {
+  let json = Python.import("json")
+
+  let swift_shell = Python.import("swift_shell")
+  let socketAndShell = swift_shell.create_shell(
+    username: _kernelCommunicator.jupyterSession.username,
+    session_id: _kernelCommunicator.jupyterSession.id,
+    key: _kernelCommunicator.jupyterSession.key)
+  _socket = socketAndShell[0]
+  _shell = socketAndShell[1]
+
+  func updateParentMessage(to parentMessage: ParentMessage) {
+    _shell.set_parent(json.loads(parentMessage.json))
+  }
+  _kernelCommunicator.handleParentMessage(updateParentMessage)
+
+  func getDisplayMessages() -> [JupyterDisplayMessage] {
```

Comment:
Swift APIs don't usually use the word "get". This function serves as a getter. The most idiomatic way is to make it a computed property.
```swift
var displayMessages: [JupyterDisplayMessage] {
  ...
}
```

Reply:
This function does have a side effect of clearing all the messages from the python buffer of messages. Would `consumeDisplayMessages` be a good name?

Reply:
Oh ok, if it's side effecting then `comsume` is a better word!